extensions/wikia/AffiliateService/js/ext.AffiliateService.js

```diff
@@ -114,13 +116,16 @@ require([
 					var availableUnits = AffiliateService.getAvailableUnits(targeting);
 
 					console.log('>', { targeting, units, availableUnits });
+
+					// add unit data to be inserted into template
+					AffiliateService.addMarker(units[0]);
 				} else {
 					console.log('No units available');
 				}
 			});
 		},
 
-		addMarker: function () {
+		addMarker: function (unit) {
```

Comment:
I think we should rename this function now, I believe it was named literally for the marker Josh was inserting.

Reply:
Renamed this to `renderUnitMarkup`.